php/classes/handlers/class-castos-handler.php

```diff
@@ -308,9 +318,10 @@ public function upload_podcast_to_podmotor( $post ) {
 
 		$this->logger->log( 'API URL', $api_url );
 
-		$featured_image_url = $this->get_featured_image( $post );
-		if ( ! empty( $featured_image_url ) ) {
-			$post_body['featured_image_url'] = $featured_image_url;
+		$cover_image_url = $this->get_cover_image_url( $post );
+		if ( ! empty( $cover_image_url ) ) {
+			// Todo: change 'featured_image_url' to 'cover_image_url' after API update
```

Comment:
Please use the @todo annotation here, it gets picked up and highlighted differently and therefore easier to find/search for